lib/components/narrative/default/default-itinerary.js

```diff
@@ -316,7 +347,7 @@ class DefaultItinerary extends NarrativeItinerary {
 const mapStateToProps = (state, ownProps) => {
   const { intl } = ownProps
   const gradationMap = state.otp.config.accessibilityScore?.gradationMap
-
+  const co2Config = state.otp.config.co2
```

Comment:
Optional chaining is probably needed here!

Reply:
Hmmm... I don't think so. If CO2 was null then it would be crashing on the previous line, right?

Reply:
Are we ever checking for the `co2` field of `config`?

Reply:
It should be checked everywhere that it's used. You did catch a place in the other PR that it wasn't being checked, though, so maybe that was causing the error? That's fixed now.